### ISSUE TITLE
Synonyms should be shown when hovering over a node in a tree

### ISSUE THREAD

**Issue by yvonneekanim** [specify/specify7#407]
Mouse overs for synonyms are not working in both directions – mouse overs showing preferred names for a synonym are working but not mouse overs for the synonym names of preferred names.

**Comment by grantfitzsimmons** [comment]
>Synonymy in trees does not appear to be working correctly in Specify 7.  When I synonymize a node in the taxon tree in my collection - in this case Phoxinus erythrogaster with Chrosomus erythrogaster - the old name appears in red to denote that it is a synonym and  a mouse over gives me the new name.  However, there is no similar functionality for the new name for which a mouseover should show all synonyms?  Did this ever work in 7?  Is there a plan to fix this?
>
> ![image](https://github.com/specify/specify7/assets/37256050/33295ed9-b47c-41fd-bc07-6eaddbdca7dc)
> ![image](https://github.com/specify/specify7/assets/37256050/df9fd93e-e837-4130-8486-87d3237ba566)

**Requested By:** Andy at KU

**Comment by CarolineDenis** [comment]
NOTES: 
- A node that has been synonimized into another node holds the fields accpetedId (new preferred id) and acceptedName (new preferred name), fields we used for the tool tip displayed on hovering. 
- The node in which we have synonimized the other one into has accpetedId and acceptedName with null value. We we need a new field on the accepted node that holds an array of synonymsName to be displayed on hover 

@maxpatiiuk, thoughts?
